packages/twenty-chrome-extension/src/options/modules/ui/input/components/TextInput.tsx

```diff
@@ -65,12 +65,15 @@ const TextInput: React.FC<TextInputProps> = ({
   placeholder,
   icon,
 }) => {
+  const inputId = useId();
```

Comment:
nice I didn't know about this one!

Reply:
This should be our default way to generated Ids in the frontend (when we don't need uuids)

Reply:
The `useId` hook remains a React hook and must follow the hooks' rules. It can't be called in response to an event. The `useId` hook is excellent for generating identifiers scoped to a React component, like HTML IDs.

Reply:
I won't say that it should be the default way for all ids but for HTML ids it's most indicated.